Automattic/wp-calypso
# Add server-friendly content sanitizer for Plugins SSR #2

#### Proposed Changes
This is a second attempt at implementing #68195. The initial PR was causing an error when visiting the /plugins page while logged in.
The issue was caused by [this commit](https://github.com/Automattic/wp-calypso/pull/68195/commits/8dda5b8f7038aa42a71cb806af6b67360d594e3c) which caches the sanitized version of the plugins. By doing this, previously browser-cached data remains unsanitized and the new flow tries to use it and fails.

This PR, adds everything from #68195 + [updates](https://github.com/Automattic/wp-calypso/pull/69280/commits/5b4c7797964b581a9d9d6d39b3e98986295caff8) the cache keys for the affected queries.

#### Testing Instructions

* Follow steps from #68195
* Enable JavaScript and log in 
* Try to run calypso under `wordpress.com` hostname, with previously cached Plugins data
* Go to /plugins or /plugins/[SITE]
* You should still be able to see the Plugins landing page with the 3 sections Paid, Editor's Pick, and Free
